.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           key: jars-{{ checksum "build.gradle" }}-{{ checksum  "Corona-Warn-App/build.gradle" }}-{{ checksum  "Server-Protocol-Buffer/build.gradle" }}
       - run:
           name: JaCoCo report
-          command: ./gradlew :Corona-Warn-App:jacocoTestReportDeviceRelease
+          command: ./gradlew :Corona-Warn-App:jacocoTestReportDeviceRelease :Corona-Warn-App:jacocoTestReportDeviceForTestersRelease
```

Comment:
This works, but I don't think it's the right approach.
With this we get unit tests on the `DeviceForTester` flavor only as side-effect of the jacocoTask.

Have you tried changing line 23? We have `quickBuild` and `quickBuildTest`, maybe we can run both there.